src/packet_sources/ipc.proto

```diff
@@ -3,26 +3,39 @@ syntax = "proto3";
 package mitmproxy.ipc;
 
 message FromRedirector {
-  oneof message {
-    PacketWithMeta packet = 1;
-  }
-}
-message PacketWithMeta {
-  bytes data = 1;
-  uint32 pid = 2;
-  optional string process_name = 3;
-}
+    message LogMessage {
+        enum LogLevel {
+            NotSet = 0;
```

Comment:
This because in [docs](https://protobuf.dev/programming-guides/proto3/#enum):
> every enum definition must contain a constant that maps to zero as its first element